data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
```

Comment:
Yes, I call them "app developers" and the other ones "core developers". I, indeed, think that it's more gratifying and more accurate (and simpler) as this is really development.